src/spec/helpers.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * this takes an object, converts stringifies it and then parses it back out.
+ * this is useful when fields are Date objects and need to be compared to a
+ * stringified version
+ * @param {*} json
+ * @returns
+ */
+module.exports.parsedJSON = (json) => {
+	return JSON.parse(JSON.stringify(json));
+};
+
+/**
+ * according to the internet this is a legit way to check if a string is
+ * an iso string
+ * @param {*} val
+ * @returns
+ */
+module.exports.isISOString = (val) => {
```

Comment:
@eacaps - can we get some test coverage for this specific method? It may be covered by the model tests, but would be good to know how it can + should handle specific edge case inputs.

Reply:
The `isISOString` and `parsedJSON` methods are only supposed to be used within the context of specs. Ideally they could be injected into mocha some way in `src/test.js` rather than specifying them in a separate file but I couldn't find a clean way to do that. In Firehawk I also moved `clearDatabase` and `userSpec` to a [helper file](https://bitbucket.org/asymmetrik/fh-webapp/src/a7b67c560e753267e87c1a00d11ad37aa71ec7ad/src/server/app/util/spec/helpers.js#lines-118) so that code is not duplicated in multiple tests.

Reply:
To add on here, I wouldn't be opposed to writing some tests to make sure these helper methods do what they say they do, but I just wanted to clarify their purpose.

Reply:
I went ahead and added some small specs for the helpers